app/components/Chat.vue.ts

```diff
@@ -29,7 +30,18 @@ export default class Chat extends Vue {
     const webview = this.$refs.chat;
     const settings = this.customizationService.getSettings();
 
-    service.getChatUrl(nightMode).then(chatUrl => this.chatUrl = chatUrl);
+    if (service instanceof YoutubeService) {
+      service.getChatUrl(nightMode).then(chatUrl => {
+        this.chatUrl = 'https://youtube.com/signin';
```

Comment:
If the user is already logged in, this should immediately redirect, right?

Reply:
Yup